sbml/iMA871/exchange.yaml

```diff
@@ -312,6 +312,9 @@ compounds:
   upper: 0.0
 - id: M_PROPALe
   reaction: '1380'
+- id: M_BIOMASS
+  compartment: C-c
```

Comment:
Also now the compartment should just be 'c' not with the C_c due to the improvements to PSAMM-import parsing.